Server/Routes/CBXRoute.h

```diff
@@ -54,5 +54,12 @@ Object containing logic for an HTTP route.
  @param block Block to execude when requests are matched to this route
  */
 + (instancetype)delete:(NSString *)path withBlock:(RequestHandler)block;
+
+/**
+ Convenience constructor for a OPTIONS route
+ @param path Route path regex
+ @param block Block to execude when requests are matched to this route
```

Comment:
```suggestion
 @param block Block to execute when requests are matched to this route
```

(Although I suppose fixing this could go into another pull request as all documentation here seems to include this typo 😄 )